kolibri/core/assets/src/views/FullScreenSidePanel/index.vue

```diff
@@ -2,53 +2,63 @@
 
   <div
     ref="sidePanel"
+    :tabindex="0"
     :class="{ 'is-rtl': isRtl, 'is-mobile': isMobile }"
     @keyup.esc="closePanel"
   >
     <transition name="side-panel">
-      <div
-        class="side-panel"
-        :style="sidePanelStyles"
+      <FocusTrap
+        :firstEl="firstFocusableEl"
```

Comment:
`firstFocusableEl` and `lastFocusableEl` elements are always `null` so lines 11-12 can be removed